### ISSUE TITLE
System_info table broken results on apple silicon m1

### ISSUE THREAD

**Issue by ar-mi** [osquery/osquery#7053]
# Bug report

### What operating system and version are you using?

MacOS Big Sur 11.2.3 on Apple MacBook Air 13" M1/8/256Gb
### What version of osquery are you using?

Osquery 4.7.0
### What steps did you take to reproduce the issue?
I did `SELECT * FROM system_info` interactive query using Fleet GUI

### What did you expect to see?
Correct info in `hardware_model` (something like MacBookAir...) and `hardware_version` columns from the query results.

### What did you see instead?
Got empty columns, like in this example:

host_hostname | cpu_brand | cpu_logical_cores | cpu_physical_cores | cpu_subtype | cpu_type | hardware_model | hardware_serial | hardware_vendor | hardware_version | physical_memory
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
***** | VirtualApple @ 2.50GHz | 8 | 8 | Intel 80486 | i486 |   | *******JQ6L4 | Apple Inc. |   | 8589934592
***** | VirtualApple @ 2.50GHz | 8 | 8 | Intel 80486 | i486 |   | *******MQ6L4 | Apple Inc. |   | 8589934592
***** | VirtualApple @ 2.50GHz | 8 | 8 | Intel 80486 | i486 |   | *******4Q6L4 | Apple Inc. |   | 8589934592
***** | VirtualApple @ 2.50GHz | 8 | 8 | Intel 80486 | i486 |   | *******FQ6L4 | Apple Inc. |   | 8589934592

**Comment by ameily** [comment]
I will be looking into this.

**Comment by zhumo** [comment]
Just doing some housekeeping... may we close this given the above merge?

**Comment by mike-myers-tob** [comment]
Yes. This was merged upstream with https://github.com/osquery/osquery/pull/7330/commits
